px-data-grid-navigation.html

```diff
@@ -22,15 +23,18 @@
       }
 
       .page-size-select {
-        display: inline-block;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        justify-content: left;
```

Comment:
There is no such option for justify-content as `left`. There are `flex-start` and `flex-end`.